### ISSUE TITLE
Add usage example on HELPERS.md

### ISSUE THREAD

**Issue by gabriel-pinheiro** [gabriel-pinheiro/bigodon#2]
## What are helpers?

Helpers are functions that you can use inside your Bigodon template to execute common tasks with ease. Check how to use them [here](https://github.com/gabriel-pinheiro/bigodon/blob/main/LANGUAGE.md#helpers) and the ones that already exist [here](https://github.com/gabriel-pinheiro/bigodon/blob/main/HELPERS.md).

## Great! How can I help

On [HELPERS.md](https://github.com/gabriel-pinheiro/bigodon/blob/main/HELPERS.md), most of the helpers are missing usage examples (click a helper's name to find out). You can click the `code` link to check how they're implemented and the `tests` one to see what's expected from each helper. If you have a good example of how a helper can be useful, please open a PR updating [HELPERS.md](https://github.com/gabriel-pinheiro/bigodon/blob/main/HELPERS.md).

**Comment by sumitkumarju** [comment]
@gabriel-pinheiro can I take it?

**Comment by gabriel-pinheiro** [comment]
Sure @sumitkumarju ! I just assigned it to you

**Comment by sumitkumarju** [comment]
@gabriel-pinheiro there are quite many helpers. Should I select some of them(4-5) and make a PR?

**Comment by gabriel-pinheiro** [comment]
@sumitkumarju yes! Feel free to pick the ones that you can think of examples where they'd be useful, don't worry about any order
